.github/workflows/reusable-audit.yml

```diff
@@ -81,4 +95,4 @@ jobs:
         run: make audit-vulnerabilities-npm
       - name: Audit production npm dependencies
         if: ${{ startsWith(matrix.ref, 'v') }}
-        run: make audit-vulnerabilities-npm ARGS="--omit dev"
+        run: make audit-npm ARGS="--omit dev"
```

Comment:
Partially reverting #969 because the new command hasn't landed in [`v0`](https://github.com/ericcornelissen/js-regex-security-scanner/tree/v0) yet.